### DISCUSSION TITLE
Github unable to find .kv (Kivy) files when selecting kvlang under Languages

### DISCUSSION THREAD

**Discussion by AnimeAllstar** [community/community#16906] [Other Feature Feedback, Questions, & Ideas] [answered]
Github is unable to find .kv (Kivy) files when selecting kvlang under Languages.
![image](https://user-images.githubusercontent.com/39626451/169261594-0ac14419-7a56-4a83-84bb-724c9973ebea.png)
![image](https://user-images.githubusercontent.com/39626451/169262042-fb254ec5-d3c3-43de-94ae-fab4730beb85.png)

I'm not sure if there are other smaller languages affected by this?

**Comment by look** [comment, marked as answer]
The code search system doesn't pick up new languages immediately (Kivy was [just added in February](https://github.com/github/linguist/pull/5788)).

I recommend joining the new [code search Technology Preview](https://cs.github.com/home), because the content there is reindexed more frequently and new languages are added more quickly.
